Update management guides - Closes #1303 (#1304)
LiskHQ/lisk-docs: modules/ROOT/pages/update/binary.adoc

----

== Backup Lisk Core
=== Backup your custom configs

> In case you created custom configurations inside the `lisk-core/` folder, make sure to copy or move them out of the `lisk-core/` folder before removing it in the next step.

Reviewer: ```suggestion
In case you created custom configurations inside the `lisk-core/` folder, make sure to copy or move them out of the `lisk-core/` folder before removing them in the next step.
```

Update the custom configuration according to the provided default configurations for Lisk Core, located in `lisk-core/config/default/`, if necessary.

> For updating to 3.0.2, most importantly, ensure to include the new property `unlockFixHeight` in the configuration.

Reviewer: ```suggestion
For updating to v3.0.2, it is highly important to ensure to the new property `unlockFixHeight` is included in the configuration.
```

> For updating to v3.0.2, it is highly important to ensure to the new property `unlockFixHeight` is included in the configuration.

Reviewer: ```suggestion
For updating to v3.0.2, it is highly important to ensure the new property `unlockFixHeight` is included in the configuration.
```
